app/segments/(team)/chooseTeam.js

```diff
@@ -99,22 +104,56 @@ function ChooseTeam() {
           >
             <Button
               onPress={async () => {
-                //temporary, should be replaced with multiple team functionality
-                await setDoc(doc(db, "teams", "1", "users", currentUserId), {
-                  name: currentUserInfo["displayName"],
-                  // hardcoded pfp string for now, add pfp upload to profile settings in future PR
-                  pfp: "",
-                  // hardcoded "player" role for now, add role selection to profile settings in future PR
-                  role: "player",
-                  uid: currentUserId,
-                  assigned_data: [],
-                  uniqueDrills: [],
-                });
-                setCurrentUserId(currentUserId);
-                await invalidateMultipleKeys(queryClient, [
-                  ["userInfo", { userId: currentUserId }],
-                ]);
-                router.replace("/");
+                let completed = false;
```

Comment:
Is this completed used to debounce the "Join team" button in case of a success?

Reply:
I was trying to prevent navigation redirect to home ("/") from triggering multiple times. I think it worked but it's pretty jank and a general debouncing PR on the button itself should also do the job.